templates/lune/lune.amp.html

```diff
@@ -18,6 +18,69 @@
   {{> ../../utils/amp-page-head.snip.html}}
   <body class="lune">
     {{> ../../components/navbar/navbar.snip.html}}
+
+    <!-- Used to achieve the clipping effect when scrolling the page -->
+    <div class="hide">
```

Comment:
You can find the gif for the effect we need to achieve at `mocks/lune/landing-page-clipping.gif`.

In particular, this `<div class="hide">`  contains the `svg` that is used in the `clip-path` CSS property (see `css/templates/lune/pages/_landing.css`(